include/rocksdb/wide_columns.h

```diff
@@ -218,4 +222,54 @@ inline bool operator!=(const PinnableWideColumns& lhs,
   return !(lhs == rhs);
 }
 
+inline PinnableWideColumns::PinnableWideColumns(const PinnableWideColumns& p)
+    : columns_(p.columns_) {
+  CopyValue(p.value_);
+}
+
+// List of PinnableWideColumns grouped by column families. statuses[i] should
+// match the status of getting columns_array[i] when GetEntity() or
+// MultiGetEntity() was called (where 0 <= i < num_column_families_)
+class GroupedPinnableWideColumns {
```

Comment:
Would `PinnableWideColumnsGroup` sound better? or please let me know if you can find a better name for this :)

Reply:
Yeah, naming stuff is always hard 😀 To avoid confusion, we might want to go with something other than "group" for the entire per-key result set though, since we already use "attribute group" for the parts.

Reply:
Taking a step back, how about modeling this the following way: we could have a simple class that represents a single attribute group/column family/"chunk"; let's call this `PinnableWideColumnsChunk` for the sake of the discussion. (BTW, this is probably not the best name! 😀) `PinnableWideColumnsChunk` would contain a `ColumnFamilyHandle` pointer, a `PinnableWideColumns` object, and a `Status`. Then, we could have another class e.g. `PinnableWideColumnsCollection`, which contains a vector of `PinnableWideColumnsChunk`s.

Using these types, the new `MultiGetEntity` could take `read_options` +  `num_keys` + `const Slice* keys` + `PinnableWideColumnsCollection* results` as parameters, where `keys` and `results` would be expected to point to `num_keys` objects as usual.

Reply:
Yeap. The model that you suggested is a lot simpler. Thank you. 

What do you think of "PinnableWideColumnsBundle" for the class name? I thought of it because it's "bundled" with status and cfh.